macros/utils/graph/get_elementary_relation.sql

```diff
@@ -1,7 +1,15 @@
 {% macro get_elementary_relation(identifier) %}
+  {% do return(adapter.dispatch("get_elementary_relation", "elementary")(identifier)) %}
```

Comment:
Why did you need to add `adapter.dispatch`?
Do you think this method will be different for different adapters?

Reply:
As I understand, in order to be able to use [this](https://docs.getdbt.com/reference/project-configs/dispatch-config) to override a macro, it must be called through the adapter object.